README.md

```diff
@@ -1 +1,47 @@
-# docker-terraform
+# docker-terraform [![CI](https://github.com/d3b-center/docker-terraform/workflows/CI/badge.svg?branch=master)](https://github.com/d3b-center/docker-terraform/actions?query=workflow%3ACI)
+
+This repository contains a collection of Nix flakes that produce container images designed to run deployments using the AWS CLI and Terraform.
+
+## Usage
+
+Via Docker:
+
+```console
+$ docker run ghcr.io/d3b-center/terraform:1.1.6 -version
+Terraform v1.1.6
+on linux_amd64
+```
+
+Or, via Docker Compose, which includes necessary bind mounts:
+
+```yml
+services:
+  terraform:
+    image: ghcr.io/d3b-center/terraform:1.1.6
+    volumes:
+      - ./:/usr/local/src
+      - $HOME/.aws:/.aws
```

Comment:
If we allow private modules, should we add ~/.ssh/ ?

Reply:
I don't think we should include that in the example as it's not universally applicable. For instance, I store my private key on my YubiKey, which SSH accesses via the SSH and GPG agent. So, I'd need to create a volume mount for the SSH agent's UNIX socket.